### DISCUSSION TITLE
Clipboards Pro

### DISCUSSION THREAD

**Discussion by kaganergovan** [elrumo/macOS_Big_Sur_icons_replacements#1211] [Icon request] [answered]
Hello community,

I'm looking for Clipboards app icon for Big Sur but I didn't find it. Could anyone clone that?

<img width="207" alt="image" src="https://user-images.githubusercontent.com/8795682/122465342-aabb3c80-cfc0-11eb-9dbf-7f174b072c25.png">

Regards,

**Comment by kaganergovan** [comment]
[App link](https://apps.apple.com/tr/app/clipboards/id1371425905?mt=12)

**Reply by OMGzilla-git** [reply]
I'm too much of an amateur for this one to look nice, but I did pull a high quality PNG icon from the store web page listing, so maybe this will help someone else make the most of it.
![1200x1200bb](https://user-images.githubusercontent.com/85633412/122467698-d5899e80-cf88-11eb-8d2b-0106564570a7.png)

**Reply by kaganergovan** [reply]
Oh, thanks for uploading high-res! :)

**Comment by blaichch** [comment]
I hope you like it!

<img width="512" alt="Clipboards Pro" src="https://user-images.githubusercontent.com/12624897/122650583-15d95000-d134-11eb-8f14-5277014affa7.png">

**Reply by OMGzilla-git** [reply]
> I hope you like it!
> 

I'm not the one who needed it but it looks great. I just have one observation. The 'C' seems off-center a bit. If you draw a vertical line from the peak of the top curve of the 'C' to the bottom, you'll see what I mean.

**Comment by blaichch** [comment]
Oh, my bad! Should be fixed now.

<img width="512" alt="Clipboards Pro" src="https://user-images.githubusercontent.com/12624897/122665278-9b95e380-d1a6-11eb-9670-0222c42929bc.png">

**Reply by OMGzilla-git** [reply]
> Oh, my bad! Should be fixed now.
> 

Mmmm... looks like you nudged it up, which actually does look a little better that way now that you did it, but I meant that it needed to be nudged to the left. Just my opinion.  :)

**Comment by blaichch** [comment, marked as answer]
The C is completely centered. But here's another version where I moved the C more to the left. I hope I got it this time 😅

<img width="512" alt="Clipboards Pro" src="https://user-images.githubusercontent.com/12624897/122685876-7fc62800-d20e-11eb-8dae-a541dd3a36d2.png">

**Reply by kaganergovan** [reply]
@blaichch thank you so much for effort. Did you send this to macosicons? It looks awesome! 🤩
